harness/determined/experimental/__init__.py

```diff
@@ -10,5 +10,6 @@
     create,
     create_trial_instance,
     test_one_batch,
-    _init_native,
+    init_native,
+    _local_execution_manager,
```

Comment:
non-blocking: should `_local_execution_manager` be named `local_execution_manager` if it's used outside of this module?

Reply:
hmm. I thought `_` is to distinguish user-facing or non-user-facing. And `_local_execution_manager` is non-user-facing.

Reply:
Talked offline. Right now, it's fine to put `_` as a prefix for non-user-facing but public module/functions. The reason is that we have a convention of only importing one module/function using only one path, which is different from what other packages typically do. Since we can't choose not to export `_local_execution_manager`, we better add a `_` to help distinguish it from other user-facing functions.